### DISCUSSION TITLE
Using MS Windows standard paths for config and data

### DISCUSSION THREAD

**Discussion by dasupradyumna** [wezterm/wezterm#4868] [Ideas]
Currently, I see that WezTerm looks for `wezterm.lua` and other helper modules under `~/.config/wezterm` or `$XDG_CONFIG_HOME/wezterm`. While this is clean on Linux systems, it ends up creating non-standard folders on Windows such as `~\.config` and `~\.local\share`.

Wouldn't it be better to use indigenous Windows folder paths such as `%APPDATA%` and `%LOCALAPPDATA%` for this purpose? I think it would improve the way wezterm interacts with the filesystem on Windows as well!

Thanks for the amazing terminal. Cheers ~

**Comment by nielpattin** [comment]
I'm using Neovim, I just want the config dir on Windows just be like `~\.config` to more simpler.
I set the environment to `$env:XDG_CONFIG_HOME = "$HOME/.config"` that is all. I always store all config there.
`%LOCALAPPDATA%` was a pain in Windows. There are hundreds of folders in there for each software I installed and some don't relate to the mean of `config` at all. :/

**Comment by wez** [comment]
I have no plans to use the appdata directories; it works against the idea that the same configuration layout can be used on multiple operating systems.  Today I can bootstrap my dotfiles from git on any OS and have the same wezterm configuration running, and I don't have to deal with remembering to use or typing in the awkward appdata paths when I'm editing my config on windows.
